projects/addon-mobile/components/index.ts

```diff
@@ -3,7 +3,6 @@ export * from '@taiga-ui/addon-mobile/components/mobile-calendar-dialog';
 export * from '@taiga-ui/addon-mobile/components/mobile-calendar-dropdown';
 export * from '@taiga-ui/addon-mobile/components/mobile-dialog';
 export * from '@taiga-ui/addon-mobile/components/pull-to-refresh';
-export * from '@taiga-ui/addon-mobile/components/sheet';
```

Comment:
We should move it to legacy, there are some features we cannot replace at the moment.

Reply:
fixed